src/api/networks.tsx

```diff
@@ -65,6 +68,7 @@ export const createClusterBridge = (
           reject(error);
           return;
         }
+        delete network.config?.parent;
```

Comment:
This probably warrants a comment above, since it's not immediately clear that the final reuqest to finalise the network creation for the cluster cannot have the `config.parent` parameter